standard/expressions.md

```diff
@@ -2150,7 +2167,7 @@ A member initializer that specifies a collection initializer after the equals si
 
 When an initializer target refers to an indexer, the arguments to the indexer shall always be evaluated exactly once. Thus, even if the arguments end up never getting used (e.g., because of an empty nested initializer), they are evaluated for their side effects.
 
-> *Example*: The following class represents a point with two coordinates:
+> *Example*: The following shows a class that represents a point with two coordinates, and the creation and initialization of an instance:
```

Comment:
Revert this change, given that the immediate example doesn't initialize an instance?

Reply:
Done